docs/src/pages/system/styled/styled.md

```diff
@@ -196,6 +196,14 @@ const MyStyledButton = (props) => (
 })
 ```
 
+### How can I use the `sx` syntax with the `styled()` utility?
```

Comment:
Should we note somewhere that there will be an overhead compared to not using `sx`?

Reply:
The overhead is the same as if you were to use the `sx` prop, no? If we state this, we should have some numbers to be more concrete.

Reply:
> The overhead is the same as if you were to use the sx prop

Yeah, we can add the numbers later then.

Reply:
I've added a note that it is the same as if the `sx` prop was used. I guess it should be enough.